android/src/main/kotlin/co/doneservices/callkeep/CallKeep.kt

```diff
@@ -409,11 +408,20 @@ class CallKeep(private val channel: MethodChannel, private var applicationContex
             launchIntent.putExtra(key, value)
         }
 
-        var answerIntent = Intent(launchIntent)
```

Comment:
After this, is the `launchIntent` still needed? I don't really understand this `PendingIntent` below.

Reply:
Yes, launchIntent gets triggered when the user clicks on the notification itself and not on the actions. When notification is clicked it gets expanded to the full screen activity for the incoming call.